packages/fxa-settings/src/pages/InlineRecoveryKeySetup/index.tsx

```diff
@@ -43,11 +44,9 @@ export const InlineRecoveryKeySetup = ({
       case 3:
         return (
           <>
-            <h1 className="text-grey-400">
-              <FtlMsg id="inline-recovery-key-setup-hint-header">
-                Security recommendation
-              </FtlMsg>
-            </h1>
+            <FtlMsg id="inline-recovery-key-setup-hint-header">
+              <HeadingPrimary>Security recommendation</HeadingPrimary>
```

Comment:
Should the heading be the same for all steps? Either "Security recommendation" or "Secure your account"?

Reply:
I remember running this one by Eduardo before - the first step is a recommendation. If they're on step 2, they're in the process of doing the thing.